### ISSUE TITLE
bigquery->bigquery copies are inefficient

### ISSUE THREAD

**Issue by seamusabshere** [faradayio/dbcrossbar#93]
they are converted to CSV, downloaded and re-uploaded. need a shortcut!

**Comment by emk** [comment]
Yup. This should be easy, so let's schedule it soon. We could also throw in a fix for #45 (losing `NOT NULL` when creating tables in BigQuery), since it's very closely related and also a pretty easy fix.

**Comment by seamusabshere** [comment]
removing faraday tag for now to register we don't need immediately

**Comment by emk** [comment]
This could be fixed in a very pretty fashion if we had #186, but there are also simpler workarounds.
